src/styles/annotator/bucket-bar.scss

```diff
@@ -28,13 +28,15 @@
     right: 0;
     pointer-events: all;
     position: absolute;
+    // Vertically center the element, which is 16px high
     margin-top: -8px;
     line-height: 1;
     height: 16px;
     width: 26px;
     -webkit-tap-highlight-color: rgba(255, 255, 255, 0);
     text-align: center;
     cursor: pointer;
+    margin-top: -8px;
```

Comment:
This property duplicates the `margin-top` rule a few lines above.

Reply:
Ah, thanks for spotting. This SASS module is...chattery :)